recipe/meta.yaml

```diff
@@ -23,18 +23,18 @@ requirements:
     - awkward ~=1.4.0
     - awkward0 ~=0.15
     - boost-histogram ~=1.1.0
-    - decaylanguage ~=0.11.0
+    - decaylanguage ~=0.12.0
     - hepstats ~=0.5.0
     - hepunits ~=2.1.0
     - hist ~=2.4.0
-    - histoprint ~=2.1.0
-    - iminuit ~=2.7.0
+    - histoprint ~=2.2.0
+    - iminuit ~=2.8.0
     - matplotlib-base >2.0.0
     - mplhep ~=0.3.0
     - numpy >=1.13.1
-    - particle ~=0.15.0
+    - particle ~=0.16.0
     - python >=3.7
-    - uproot ~=4.0.0
+    - uproot 4.0.*
```

Comment:
```suggestion
    - uproot ~=4.0.0
```

Reply:
@chrisburr, I reverted because the conda release needs to be in phase with the pip release. Yes I now see that uproo4 moved in the meantime to version 4.1 and I had not noticed it in my list of update dependencies, but that update will be for a future release, then.